### ISSUE TITLE
Update `warranty` field in Return

### ISSUE THREAD

**Issue by Lance-Inman** [dpsrr/ETA#18]
- Change Return 'warranty' to display as "In Warranty Period"
- Change Return 'warranty' to be editable
- Add section in Traveler Creator to specify "In Warranty Period"

**Comment by Lance-Inman** [comment]
I pasted this in Discord a while back but it was late at night so it probably got buried.

**Comment by AtlasHands** [comment]
Added traveler creator "In Warranty"  Do you think Period is needed as a descriptor?

**Comment by Lance-Inman** [comment]
> Added traveler creator "In Warranty" Do you think Period is needed as a descriptor?

No, I think when we eventually have an object for each unit, the warranty period will be part of its data.
